README.md

```diff
@@ -1,5 +1,53 @@
-# ttc-charts
+# Trending Topics Campaigns (TTC) - Helm Charts
 
-Not a GitHub Pages repo - clone the repo to use the chart.
+For documentation, please refer to [TTC Docs](https://github.com/Activiti/ttc-docs)
+
+Note: This is not a GitHub Pages repo. Please clone the repo to use the chart.
 
 Currently only the backend of the example as the UI is not yet fully parameterised. To be used for running ttc-acceptance-tests
+
+## Notes for using Docker for Desktop (with local k8s cluster)
+
+Here are some notes that may help you to run the example locally using Docker for Desktop with a local k8s cluster. 
+Note: other options include using minikube or installing onto a remote k8s cluster running in your favourite cloud provider, eg. AWS, Azure or GCP.
+
+1. Change to ttc-example directory
+
+```
+cd /charts/ttc-example
+```
+
+2. Find your host ip
+
+```
+kubectl describe node docker-for-desktop  | grep InternalIP | awk '{print $2}'
```

Comment:
TODO 1: the internal IP is not the right one. What's the easiest way to get the host ... via ifconfig ?

TODO 2: Also, might be easier to add a mapping to /etc/hosts to allow it ip to be changed in one place.